figshare/figshare.py

```diff
@@ -0,0 +1,111 @@
+import sys
+import os
+import hashlib
+import json
+import requests
+
+from requests.exceptions import HTTPError
+
+
+def issue_request(method, url, headers, data=None, binary=False):
```

Comment:
Let's add [docstrings](http://docs.python-guide.org/en/latest/writing/documentation/) describing what the function does and what the parameters are.